go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	k8s.io/component-base v0.0.0
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.2.0
-	k8s.io/kubernetes v1.19.1
+	k8s.io/kubernetes v1.19.3
```

Comment:
This should probably be `v0.0.0-20201023141757-9e8ad8ce9d8a` until we update this to `v1.1.9.4` in a follow-up PR

Reply:
updated! I also re-verify that the node registration worked after updating :)